Makefile

```diff
@@ -1,7 +1,9 @@
 NAME      := mpifileutils
 SRC_EXT   := gz
 REPO_NAME := mpifileutils-pkg
-GIT_SHORT := bdc379b
+GIT_SHORT := 49b04c1
+PKG_GIT_COMMIT := 49b04c1
+GITHUB_PROJECT := hpc/$(NAME)
 
 BUILD_DEFINES := --define "git_short $(GIT_SHORT)"
```

Comment:
This should be removed from this file.

Reply:
Please don't resolve conversations that I start.

Reply:
@brianjmurrell I had it removed yesterday and the build was failing. can try again later but as soon as I removed these suggestions the build passed.